src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1242,6 +1242,9 @@ private function registerCacheConfiguration(array $config, ContainerBuilder $con
         $container->getDefinition('cache.adapter.system')->replaceArgument(2, $version);
         $container->getDefinition('cache.adapter.filesystem')->replaceArgument(2, $config['directory']);
 
+        if (isset($config['prefix_seed'])) {
+            $container->setParameter('cache.prefix.seed', $config['prefix_seed']);
```

Comment:
Guess to avoid creating params into the wild and making things less implicit. Im fine with it though.. just curious :)

This mix&matches `framework.cache.prefix_seed` and `%cache.prefix.seed%`, they're the same. In apps this often shows a code smell, as people tend to use params as some global config array.